lib/resources/example_json/soarian_examples_link_extension.rb

```diff
@@ -0,0 +1,18 @@
+module Cerner
```

Comment:
```suggestion
# frozen_string_literal: true

module Cerner
  module Resources
    SOARIAN_LINK_EXTENSION ||= {
      "type": "searchset",
      "link": [
        {
          "relation": "self",
          "url": "http://fhir-myrecord.sandboxcerner.com/dstu2/Observation?patientId=2A39DA0B-61D8-4F95-BE00-9CA30EE01819&_format=json&page=2"
        },
        {
          "relation": "next",
          "url": "http://fhir-myrecord.sandboxcerner.com/dstu2/Observation?patientId=2A39DA0B-61D8-4F95-BE00-9CA30EE01819&_format=json&page=3"
        }
      ]
    }
  end
end
```

Reply:
not needed

Reply:
"module Cerner" is the pattern followed in all the files and we are adhering to that.

Reply:
Changes are present here. https://github.com/cerner/fhir.cerner.com/pull/232/commits/416d911dabbb8e9fdd2602dbf306ae20ee43e10d